.github/workflows/ompi_mpi4py.yaml

```diff
@@ -0,0 +1,40 @@
+name: GitHub Action CI
+
+on: [pull_request]
+
+env:
+  MPI4PY_VERSION: 3.1.5
+jobs:
+  mpi4py:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Install dependencies
+      run: |
+        sudo apt update
+        sudo apt install -y --no-install-recommends wget lsb-core software-properties-common curl python3-pip
+        pip3 install cython
+    - uses: actions/checkout@v3
+      with:
+            submodules: recursive
+    - name: Build Open MPI
+      run: |
+        ./autogen.pl
+        ./configure --prefix=${GITHUB_WORKSPACE}/install --disable-oshmem --disable-mpi-fortran
+        make -j install
+    - name: Checkout mpi4py
+      uses: actions/checkout@v4
+      with:
+        repository: "mpi4py/mpi4py"
```

Comment:
This is always going to get the tip of the default branch, right?  Should we, instead, be testing with known stable releases?

Reply:
> Should we, instead, be testing with known stable releases?

https://github.com/open-mpi/ompi/pull/12217#issuecomment-1878641624

Reply:
@dalcinl 👍